### PR TITLE
Remove variables from exception messages

### PR DESCRIPTION
**Pull request by fvovan**
Rather move them to exception's `metaInfo`

### CODE REVIEW

**Comment by fvovan** [comment]
@tomaszdurka  please review.
should we make changes to guides? (section PHP - exceptions )

**Comment by tomaszdurka** [comment]
138 files wot!

**Comment by fvovan** [comment]
yes.. it probably could be splitted to several PRs but in the end we'll have to adjust them all

File: library/CM/Action/Abstract.php [hunk 1]
```diff
@@ -452,7 +455,7 @@ public static function getNameByType($type) {
     public static function getVerbByVerbName($name) {
         $actionVerbs = self::_getConfig()->verbs;
         if (!array_key_exists($name, $actionVerbs)) {
-            throw new CM_Exception_Invalid('Action `' . $name . '` does not exist!');
+            throw new CM_Exception_Invalid('Action does not exist!', null, ['actionName' => $name]);
```
Comment (tomaszdurka) [inline library/CM/Action/Abstract.php]: I love the drama in the exclamation marks :) !!!

File: library/CM/App.php [hunk 2]
```diff
@@ -199,7 +199,10 @@ public function _getUpdateScriptPath($version, $moduleName = null) {
         }
         $file = new CM_File($path . 'resources/db/update/' . $version . '.php');
         if (!$file->exists()) {
-            throw new CM_Exception_Invalid('Update script `' . $version . '` does not exist for `' . $moduleName . '` namespace.');
+            throw new CM_Exception_Invalid('Update script does not exist for module namespace.', null, [
```
Comment (tomaszdurka) [inline library/CM/App.php]: Could be just `Update script does not exist` (pain-in-the-ass-mode)

**Comment by tomaszdurka** [comment]
Ok after reviewing first 20 files, decided to trust it's good if tests are passing.
@fvovan Are there any changes except exception messages changes (plus test adjustments) done here?

Please merge upstream and let's merge this one.

**Pull request merged**